models/artifacts.go

```diff
@@ -0,0 +1,24 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// Artifact represents the artifacts table
+type Artifact struct {
+	ID            uuid.UUID  `json:"id"`
+	CheckID       uuid.UUID  `json:"check_id,omitempty"`
+	CheckTime     time.Time  `json:"check_time,omitempty" time_format:"postgres_timestamp"`
+	PlaybookRunID uuid.UUID  `json:"playbook_run_id,omitempty"`
```

Comment:
Maybe we should use join tables instead e.g. check_artifacts, playbook_run_artifacts

Reply:
as discussed we can handle this later